src/test/kotlin/org/openmbee/flexo/mms/LDAPAuthenticationTest.kt

```diff
@@ -134,6 +136,47 @@ class LDAPAuthenticationTest {
         }
     }
 
+    @Test
+    fun testCheckLogin() = testApplication {
+        environment {
+            config = MapApplicationConfig(
```

Comment:
This config is the same as what is under `testGetLogin`, could there be a way to combine them into one and have it be shared between the two?